dev/test/bulk-writer.ts

```diff
@@ -510,4 +513,84 @@ describe('BulkWriter', () => {
     await flush2;
     return bulkWriter.close();
   });
+
+  describe('if bulkCommit() fails', async () => {
+    it('flush() should not fail', async () => {
```

Comment:
Since this entire test suite uses a different behavior for `instantiateInstance`, you can simply overwrite it (or use a diffent function name).

https://gist.github.com/schmidt-sebastian/0724808e4f45f356602085cffe90cea9

Reply:
thanks for the gist!